doc/userguide/rules/tls-keywords.rst

```diff
@@ -121,6 +121,19 @@ Examples::
 to use the previous name, but it's recommended that rules be converted to use
 the new name.
 
+tls.subjectaltname
+------------------
+
+Match TLS/SSL Subject Alternative Name field.
```

Comment:
Seeing the PR with the multi-buffer tests and the conversation that originated that addition, I think it'd be good to add here that this supports multiple buffers, and maybe add an example? :)

Reply:
Sorry for keeping adding to this but just got curious: does this support negated content?

Reply:
likely but https://redmine.openinfosecfoundation.org/issues/2224